data-tools/openai.ts

```
@@ -420,3 +420,106 @@ Keep your response brief and structured according to the function schema.`
 		};
 	}
 }
+
+/**
+ * Function to call OpenAI with a JSON schema for structured responses
+ */
+export async function callOpenAIWithSchema<T>({
```

Biome (1.9.4)

[error] Shouldn't redeclare 'callOpenAIWithSchema'. Consider to delete it or rename it.

'callOpenAIWithSchema' is defined here:


(lint/suspicious/noRedeclare)